frontend/src/pages/WorkspaceTeam/WorkspaceTeam.tsx

```diff
@@ -1,26 +1,62 @@
+import Tabs from '@components/Tabs/Tabs'
 import { useGetWorkspaceAdminsQuery } from '@graph/hooks'
 import { AdminRole, WorkspaceAdminRole } from '@graph/schemas'
-import { Box } from '@highlight-run/ui'
+import { Badge, Box, IconSolidUserAdd, Stack, Text } from '@highlight-run/ui'
 import AllMembers from '@pages/WorkspaceTeam/components/AllMembers'
 import AutoJoinForm from '@pages/WorkspaceTeam/components/AutoJoinForm'
 import InviteMemberModal from '@pages/WorkspaceTeam/components/InviteMemberModal'
 import { Authorization } from '@util/authorization/authorization'
 import { useParams } from '@util/react-router/useParams'
 import React from 'react'
+import { useNavigate } from 'react-router-dom'
 import { useToggle } from 'react-use'
 
 import Button from '../../components/Button/Button/Button'
 import layoutStyles from '../../components/layout/LeadAlignLayout.module.scss'
 import styles from './WorkspaceTeam.module.scss'
 
+type MemberKeyType = 'members' | 'invites'
+
 const WorkspaceTeam = () => {
-	const { workspace_id } = useParams<{ workspace_id: string }>()
+	const { workspace_id, member_tab_key = 'members' } = useParams<{
+		workspace_id: string
+		member_tab_key?: MemberKeyType
+	}>()
 	const { data, error, loading } = useGetWorkspaceAdminsQuery({
 		variables: { workspace_id: workspace_id! },
 		skip: !workspace_id,
 	})
+	const navigate = useNavigate()
 	const [showModal, toggleShowModal] = useToggle(false)
 
+	const TabContentContainer = ({ children }: { children: any }) => (
+		<Box mt="8">
+			<div className={styles.tabTitleContainer}>
```

Comment:
Nit, but I don't see anything being applied in the `tabTitleContainer` CSS properties that couldn't be applied directly through `Box` or `Stack`. e.g.

.tabTitleContainer {
	align-items: center;
	display: flex;
	justify-content: space-between;
	margin-bottom: var(--size-xSmall);
}


```tsx
<Stack
  mb="8"
  align="center"
  justify="space-between"
>
 ...
</Stack>
```

In general, we try to leverage the `Box` component for applying basic styles like this. It has access to all of our CSS utility classes (similar to something like Tailwind, but created with a tool called Sprinkles from [vanilla-extract](https://vanilla-extract.style/)) via its props. It's inevitable we will have to write some custom styles once in a while, but we're trying to move away from SCSS modules entirely + leverage the utility classes via `Box` and `Stack` whenever possible.

There is [a tech talk on some of this](https://www.notion.so/runhighlight/Building-A-Component-Library-e0b9e29f381342328532686a49c11780?pvs=4), but it's probably worth going over in more detail the next time we pair!